src/views/UserSettingsView.java

```diff
@@ -286,6 +303,11 @@ public void propertyChange(PropertyChangeEvent evt) {
             RemoveFriendState removeFriendState = (RemoveFriendState) evt.getNewValue();
             JOptionPane.showMessageDialog(this, "Friend " + removeFriendState.getFriendName() + " removed successfully!", "Success", JOptionPane.INFORMATION_MESSAGE);
             refreshFriends();
+        } else if ("exportCalendarSuccess".equals(evt.getPropertyName())) {
+            JOptionPane.showMessageDialog(this, "User calendar is successfully exported to the Downloads Directory of LinkUp", "Export Success", JOptionPane.INFORMATION_MESSAGE);
```

Comment:
![image](https://github.com/user-attachments/assets/b53dafee-9ea4-4529-be9b-4048e41efb77)

The exports are no longer in the downloads directory, could you change this message to specify the correct directory now? (`CalendarExports`)